pkg/kor/networkpolicies_test.go

```diff
@@ -0,0 +1,143 @@
+package kor
+
+import (
+	"context"
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Please align to `v1` as seen in other resource tests and refactor where needed.